src/prpy/util.py

```diff
@@ -1326,7 +1326,7 @@ def ComputeGeodesicUnitTiming(traj, env=None, alpha=1.0):
     return new_traj
 
 
-def CheckJointLimits(robot, q, deterministic=None):
+def CheckJointLimits(robot, q, hard_limits, deterministic=None):
```

Comment:
Thanks! @mkoval @cdellin I made a first pass at a Meta Planner in `...planning/base.py`. Can you glance over it and tell me if it seems to be doing the right thing? I'll refine accordingly.